content/attacks/posts/2021-06-23-Haven-Protocol.md

```diff
@@ -6,30 +6,30 @@ title: "Haven Protocol Hack"
 
 ## Summary
 
-Haven Protocol experienced a series of exploits that resulted in financial losses and raised concerns about the security of the platform. The exploits targeted various aspects of the protocol, including miner rewards, transaction validation, and conversion fees.
+Haven Protocol experienced a series of exploits that resulted in financial losses. The attacker [manipulated](https://havenprotocol.org/wp-content/uploads/2021/07/Technical-Overview-of-June-2021-Exploits.pdf) their transaction by modifying the proof-of-value and burnt/mint checks, setting the amount burnt/mint to 0. Since multiplying 0 by any value results in 0, the validation passed, allowing the attacker to proceed. By manipulating the output values, the attacker was able to mint an arbitrary amount.

```

Comment:
There is no such thing as 'manipulating transaction'. Also, source is not accessible by now.